lib/components/typography/storybook/index.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Meta, Story } from "@storybook/react/types-6-0";
+import { Typography } from "../src";
+
+export default {
+  title: "Typography",
+} as Meta;
+
+const Template: Story = () => (
+  <div>
+    <Typography.H1>Hello, this is a H1</Typography.H1>
+    <Typography.H2>Hello, this is a H2</Typography.H2>
+    <Typography.H3>Hello, this is a H3</Typography.H3>
+    <Typography.H4>Hello, this is a H4</Typography.H4>
+    <Typography.H5>Hello, this is a H5</Typography.H5>
+    <Typography.H6>Hello, this is a H6</Typography.H6>
+    <div>
```

Comment:
As the body is just using a span element under the hood, we have to wrap it in a div if we want it to break on the page nicely